IDBOOKAPI/api/viewsets.py

```
@@ -996,93 +1173,147 @@ class EnquiryViewSet(viewsets.ModelViewSet):
-    filterset_fields = ['id', 'title', 'active', ]
+    filterset_fields = [
+        "id",
+        "title",
+        "active",
+    ]
     # http_method_names = ['get', 'post', 'put', 'patch', 'head', 'options']
 
 
 class PaymentGatewayViewSet(viewsets.ModelViewSet):
-    queryset = PaymentGateway.objects.filter(active=True).order_by('created')
+    queryset = PaymentGateway.objects.filter(active=True).order_by("created")
```

Ruff (0.14.8)

`PaymentGateway` may be undefined, or defined from star imports

(F405)